util/src/test/resources/org/killbill/billing/util/ddl_test.sql

```diff
@@ -65,12 +65,13 @@ CREATE TABLE bundles (
     tenant_record_id bigint /*! unsigned */ not null default 0,
     PRIMARY KEY(record_id)
 ) /*! CHARACTER SET utf8 COLLATE utf8_bin */;
-
+CREATE UNIQUE INDEX bundles_external_key ON bundles(external_key, tenant_record_id);
```

Comment:
I suppose this is because of [this](https://github.com/killbill/killbill/blob/master/subscription/src/main/resources/org/killbill/billing/subscription/ddl.sql#L125) ?